src/commands/synthetics/interfaces.ts

```diff
@@ -282,6 +282,8 @@ export interface Test extends ServerTest {
   suite?: string
 }
 
+export type LocalTestDefinition = Omit<ServerTest, 'public_id'>
```

Comment:
Let's remove all the irrelevant fields. From the [RFC](https://docs.google.com/document/d/16AwC0moUzRYjXfSB0SrTte3fhvenwbhqXCjiXIViNmQ/edit?pli=1&tab=t.0#heading=h.pf3svqg4q2o8), that's:
`message`, `monitor_id`, `options.tick_every`, `options.min_failure_duration`, `options.min_location_failed`, `options.monitor_name`, `options.monitor_priority`, `options.monitor_options`, `options.scheduling`, `status`, `tags`.

Given that it might be quite convoluted to remove all the `options.`, it might be preferable to start from an LTD type, and build the Server type around it, no?